CHANGELOG.md

```diff
@@ -2,7 +2,25 @@
 
 List of the most important changes for each release.
 
-## 0.12.y
+0.12.y
+------
+
+0.12.3
+------
+
+Changed or fixed
```

Comment:
This is still in RST format, no? We probably need to migrate it over to MD.

Reply:
I can't tell. I did change the line under `0.12.3` from `======` to `-----`. I think the rest of the doc is MD syntax.

Reply:
oh, i see the 0.12.3 stuff is still RST